07LessonSeven/in-class-project/index.html

```diff
@@ -2,9 +2,74 @@
 <html>
 
 <head>
-    <title>Template</title>
+    <title>NASA</title>
+    <link href="css/my-style.css" rel="stylesheet" />
 </head>
 
-<body></body>
+<body>
+    <!-- NASA background  -->
+    <section class="nasa-background">
+        <h1>NASA</h1>
+    </section>
 
-</html>
\ No newline at end of file
+    <!-- Gray section below NASA background -->
+    <section class="contactsection">
+

```

Comment:
since you classed all the divs below contactsection with subsection, you could just go through the section to make the changes if you don't want to class these guys. #lazy